src/dev/build/tasks/bin/scripts/kibana.bat

```diff
@@ -16,11 +16,11 @@ If Not Exist "%NODE%" (
 
 set CONFIG_DIR=%KBN_PATH_CONF%
 If [%KBN_PATH_CONF%] == [] (
-  set CONFIG_DIR=%DIR%\config
```

Comment:
@jbudz I think we should also do the same for `kibana-keystore.bat` and `kibana-plugin.bat`

Reply:
Good catch.  Pushed up with d02d570.